src/routes/home.tsx

```diff
@@ -0,0 +1,8 @@
+import Home from '../components/Home/Home'
+
+export default {
+  applicant: {
```

Comment:
Line 4 should be `home:`

After our conversation I'm a little curious if it'd all work fine just exporting an un-named object. At this moment, I believe line 4's functionality is purely semantic.